### ISSUE TITLE
[BUG] PPL example on website not working

### ISSUE THREAD

**Issue by wjonkerhulst** [opensearch-project/sql#194]
**Describe the bug**
The example on https://opensearch.org/docs/search-plugins/ppl/index/ is not working.

**To Reproduce**
1. Go to the website https://opensearch.org/docs/search-plugins/ppl/index/
2. Copy and paste the code in de Quick Start section to create the accounts index on your cluster
3. In your cluster go to de Query Workbench, select PPL in de right corner
4. Execute the PPL query that is suggested on https://opensearch.org/docs/search-plugins/ppl/index/

**Expected behavior**
Query result.

**Plugins**
Opensearch distro 7.10

**Screenshots**
![afbeelding](https://user-images.githubusercontent.com/54948500/129537334-560aadaa-d9a5-4e92-b039-be01156ab0ab.png)

**Host/Environment (please complete the following information):**
 - OS: AWS Elasticsearch
 - Version 7.10

**Comment by CEHENKLE** [comment]
Transferring to the SQL team.  If I've move this in error, please @ me and I'll move it back :)

**Comment by chloe-zh** [comment]
Hi @wjonkerhulst , I tried the same steps in the documentation, but was not able to reproduce it. Could you try to execute in dev tools by using the following command to see what the error is returned?
```
POST _opendistro/_sql
{
  "query":"search source=accounts | where age > 18 | fields firstname, lastname"
}
```

**Comment by wjonkerhulst** [comment]
@chloe-zh 
![afbeelding](https://user-images.githubusercontent.com/54948500/132171338-79394eae-fc90-42d9-a5be-f29f13f66bc6.png)

**Comment by chloe-zh** [comment]
My bad, the endpoint should be _opendistro/_ppl

**Comment by wjonkerhulst** [comment]
@chloe-zh No problem. The PPL endpoint works.

![afbeelding](https://user-images.githubusercontent.com/54948500/132176321-97608d38-8a06-4f6c-8cfa-53abd0273904.png)

**Comment by wjonkerhulst** [comment]
@chloe-zh I think I have found the problem.

A single line query works:
![afbeelding](https://user-images.githubusercontent.com/54948500/132176530-88109be5-4b16-4ba3-a48c-eb1e57e1b385.png)

But when I copy and paste the example it is multi-line and the query stops working.
![afbeelding](https://user-images.githubusercontent.com/54948500/132176642-cec01c6c-65af-4f40-ac83-48e4f585c5de.png)

**Comment by camAtGitHub** [comment]
Hello @chloe-zh the issue is that 'Query Workbench' is not escaping quotes etc.
Attempting to run `search source=accounts | where gender="F"` results in: `Events: no response, this query is not runnable.`  
However running `search source=accounts | where gender=\"F\"` returns the expected data.  
The frustrating part is that documentation is either incorrect or an escaping bug has been introduced.

Thanks

**Comment by joshuali925** [comment]
Hi, looks like there are two issues:
1. line breaks were not handled correctly for PPL in workbench, causing multiline query to fail
2. quotes are not properly escaped

Both will be fixed in #305, thanks for reporting them!
